docs/migration/migrate_6_0.asciidoc

```diff
@@ -55,6 +55,14 @@ This is no longer the case. Now, only commas are a valid query separator: e.g. `
 
 [float]
 === Time-interval based index patterns are no longer supported
-*Details:*  Starting in Kibana 6.0.0 we removed the ability to create index patterns that use a date-pattern and interval to identify Elasticsearch indices. Index patterns must now use wildcards which are more performant in most cases.
+*Details:* Starting in Kibana 6.0.0 we removed the ability to create index patterns that use a date-pattern and interval to identify Elasticsearch indices. Index patterns must now use wildcards which are more performant in most cases.
 
-*Impact:* Existing index patterns and saved objects will continue to function without issue, and in a subsequent release we will provide utilities to migrate your index patterns/saved objects.
\ No newline at end of file
+*Impact:* Existing index patterns and saved objects will continue to function without issue, and in a subsequent release we will provide utilities to migrate your index patterns/saved objects.
+
+
+[float]
+=== CSRF protection requires `kbn-xsrf` HTTP header to be presented in every DELETE/POST/PUT request
```

Comment:
Just my attempt to describe a breaking change, happy to change it to better wording :)